Vagrantfile

```diff
@@ -1,19 +1,22 @@
-# -*- mode: ruby -*-
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/zesty64"
-  config.vm.network "forwarded_port", guest: 4000, host: 4000, host_ip: "127.0.0.1"
-  config.ssh.forward_agent = true
+  config.vm.box = "pgrunwald/elixir-phoenix-ubuntu-trusty64"
+  config.vm.network "forwarded_port", guest: 4000, host: 1234, host_ip: "127.0.0.1"
```

Comment:
I'd leave the host port on `4000` it doesn't matter if you are running in Vagrant or locally, `localhost:4000` will do the right thing.